stubs/config/kafka.stub

```diff
@@ -3,11 +3,30 @@
 }}}
 import env from '#start/env'
 
+const kafkaSecurityProtocol = env.get('KAFKA_SECURITY_PROTOCOL', 'PLAIN')
+const useSSL = kafkaSecurityProtocol === 'SSL'
+const useSasl = env.get('KAFKA_SASL_MECHANISM') !== undefined
+
+const saslOptions = useSasl
```

Comment:
not sure we want this in the default stub. I’d opt for getting folks going fast in a dev env. But we could leave this commented out so folks know they have this path for a live env

Reply:
Mmm, maybe. Idk.